### PR TITLE
Simple fix for Login URL formatting of double slash

### PR DESCRIPTION
**Pull request by dghelm**
# PULL REQUEST

## Overview
Currently, URLs for login and signup are formatted to look like: `https://account.skynetpro.net//auth/login`, using a double slash.

image

I'm unsure if there is code elsewhere to ensure that accountsUrl ends with a `/`, but I doubt the react app is the place for that logic anyways.

### CODE REVIEW

**Review by m-cat — APPROVED** [review]
> I'm unsure if there is code elsewhere to ensure that accountsUrl ends with a /, but I doubt the react app is the place for that logic anyways.

Sounds like something we should put in the (non-existent) `skynet-utils` repo I've been dreaming about.

**Comment by dghelm** [comment]
Ah, nice Karol. I will copy that trick in the future.

File: packages/website/src/components/Uploader/Uploader.js [hunk 1]
```diff
@@ -22,11 +22,11 @@ const getRootDirectory = (file) => {
 };
 
 const RegistrationLink = () => {
-  const accountsUrl = useAccountsUrl();
+  const createAccountsUrl = useAccountsUrl();
```
Comment (m-cat) [inline packages/website/src/components/Uploader/Uploader.js:25]: Why is this function being assigned to a const? And why are we overriding the existing `createAccountsUrl` function? I don't know how React works but this looks really weird.

File: packages/website/src/services/useAccountsUrl.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import skynetClient from "./skynetClient";
 
 export default function useAccountsUrl() {
   const [url, setUrl] = React.useState("");
+  const createAccountsUrl = React.useCallback((path = "") => url && new URL(path, url).toString(), [url]);
```
Comment (m-cat) [inline packages/website/src/services/useAccountsUrl.js:6]: Can we unit test this?

File: packages/website/src/components/Uploader/Uploader.js (hunk 1 above)
Comment (m-cat) [reply packages/website/src/components/Uploader/Uploader.js:25]: I see that `createAccountsUrl` is actually returned from the `useAccountsUrl` function. It still looks weird. I also don't understand why we are assigning this to a variable.

**Review by m-cat — CHANGES_REQUESTED** [review]
- I don't really understand the new changes introduced by @kwypchlo .
- Would be awesome if the changes were unit-tested. If there is no unit-testing suite we need to set it up.

- f/u: If we don't have it yet we should enact a baseline test coverage. We don't need full coverage but new changes should not decrease coverage.

File: packages/website/src/services/useAccountsUrl.js (hunk 2 above)
Comment (m-cat) [reply packages/website/src/services/useAccountsUrl.js:6]: I see that there were a couple additional commits to fix bugs which is why we need automated tests for functions like this. It's easy to get logic wrong or miss edge cases.

File: packages/website/src/components/Uploader/Uploader.js (hunk 1 above)
Comment (kwypchlo) [reply packages/website/src/components/Uploader/Uploader.js:25]: `useAccountsUrl` is a custom react hook https://reactjs.org/docs/hooks-intro.html and that's why it has to be assigned inside of the component

File: packages/website/src/services/useAccountsUrl.js (hunk 2 above)
Comment (kwypchlo) [reply packages/website/src/services/useAccountsUrl.js:6]: I skipped on unit testing components due to time constraints, maybe now that we have one more frontend dev on board we will be able to do better in this regard.

**Review by peterjan — APPROVED** [review]
LGTM 👍

**Review by m-cat — APPROVED** [review]
